Source/JavaScriptCore/heap/Strong.h

```diff
@@ -144,6 +160,10 @@ template <typename T, ShouldStrongDestructorGrabLock shouldStrongDestructorGrabL
         HandleSet::heapFor(slot())->template writeBarrier<std::is_base_of_v<JSCell, T>>(slot(), value);
         *slot() = value;
     }
+
+#if ASSERT_ENABLED
+    uintptr_t strongID;
```

Comment:
Are `Strong` handles unique? Could you just use the address of `this` as the token instead?

Reply:
They get moved, unfortunately. I originally did this.

Reply:
Maybe I could investigate why Vector wasn't running my destructor, I just did this as an easy solution.

Reply:
I did investigate this, the issue was that templated classes can't really export static members to other modules. Fixed!